data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py

```diff
@@ -0,0 +1,318 @@
+             (drug_concept_id is null or drug_concept_id = 0)) as n_records_to_drop,
+     'drug_exposure' as table_to_drop_from
+    FROM `{project_id}.{clean_dataset}.drug_exposure`
+
+    UNION ALL
+
+    SELECT
+     COUNTIF((device_source_concept_id is null or device_source_concept_id = 0) AND
+             (device_concept_id is null or device_concept_id = 0)) as n_records_to_drop,
+     'device_exposure' as table_to_drop_from
+    FROM `{project_id}.{clean_dataset}.device_exposure`
+
+    UNION ALL
+
+    SELECT
+     COUNTIF((observation_source_concept_id is null or observation_source_concept_id = 0) AND
+             (observation_concept_id is null or observation_concept_id = 0)) as n_records_to_drop,
+     'observation' as table_to_drop_from
+    FROM `{project_id}.{clean_dataset}.observation`
+
+    UNION ALL
+
+    SELECT
+     COUNTIF((measurement_source_concept_id is null or measurement_source_concept_id = 0) AND
+             (measurement_concept_id is null or measurement_concept_id = 0)) as n_records_to_drop,
+     'measurement' as table_to_drop_from
+    FROM `{project_id}.{clean_dataset}.measurement`
+
+#    UNION ALL
+#
+#    SELECT
+#     COUNTIF((cause_source_concept_id is null or cause_source_concept_id = 0) AND
+#             (cause_concept_id is null or cause_concept_id = 0)) as n_records_to_drop,
+#     'death' as table_to_drop_from
+#    FROM `{project_id}.{clean_dataset}.death`
+)
+SELECT coalesce(n_records_to_drop, 0) as n_violations,
+ coalesce(table_to_drop_from, 'NO VIOLATORS') as violating_table
+FROM should_be_dropped
+WHERE n_records_to_drop > 0
+'''
+execute(client, query)
+
+# # Simplified Clean Weight Check
+# Not checking all the clean weight bells and whistles.  Just checking to make sure no site
+# has submitted rows with measurement_concept_id in (3025315, 3013762, 3023166).  If any
+# of these values are greater than 0, a portion of the clean_height_weight rule failed.
+# Based on query defiend in line 561 of clean_height_weight.py
+# # IGNORE THESE RESULTS.  BAD ANALYSIS. 
+
+query = f'''
+SELECT 
+COUNTIF(m.measurement_concept_id = 3025315) as n_3025315,
+COUNTIF(m.measurement_concept_id = 3013762) as n_3013762,
+COUNTIF(m.measurement_concept_id = 3023166) as n_3023166
+FROM `{project_id}.{clean_dataset}.measurement` as m
+LEFT JOIN `{project_id}.{clean_dataset}.measurement_ext` as me
+USING (measurement_id)
+WHERE not REGEXP_CONTAINS(me.src_id, r'(?i)PPI/PM')
+'''
+#execute(client, query)
+
+# # Simplified Clean Height Check
+# Not checking all the clean height bells and whistles.  Just checking to make sure no site
+# has submitted rows with measurement_concept_id in (3036277, 3023540, 3019171).  If any
+# of these values are greater than 0, a portion of the clean_height_weight rule failed.
+# Based on query defined in line 249 of clean_height_weight.py.
+# # IGNORE THESE RESULTS.  BAD ANALYSIS. 
+
+query = f'''
+SELECT 
+COUNTIF(m.measurement_concept_id = 3036277) as n_3036277,
```

Comment:
@ksdkalluri I would recommend getting rid of these two queries entirely.  They are dead code and do not function as intended.  They are also blatantly skipped.

Reply:
Done Updated!